### PR TITLE
feat: display EDS icon in tabs view of attribute-selector

### LINKED ISSUE #168: feature(attribute-selector): add support for specifying eds icons
Icons to show in the sidebar and perhaps tabs (if it looks good)

### PR DESCRIPTION
**Pull request by KristianKjerstad**
## What does this pull request change?
add option to display EDS icon in tabs selector 
image

image

## Issues related to this change
closes #168

### CODE REVIEW

File: packages/dm-core-plugins/src/attribute-selector/Icon.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import { Icon } from '@equinor/eds-core-react'
+import {
```
Comment (soofstad) [inline packages/dm-core-plugins/src/attribute-selector/Icon.ts:2]: Could this possible be solved with a dynamic import? Making all EDS icons available without typing them out

**Review by soofstad — APPROVED** [review]

File: packages/dm-core-plugins/src/attribute-selector/Icon.ts (hunk 1 above)
Comment (KristianKjerstad) [reply packages/dm-core-plugins/src/attribute-selector/Icon.ts:2]: [will check](https://equinor.slack.com/archives/CJT20H1B9/p1681886587286679) before completing this PR :)

File: example/docker-compose.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ version: "3.4"
 
 services:
   dmss:
-    image: datamodelingtool.azurecr.io/dmss:latest
+    image: datamodelingtool.azurecr.io/dmss:v1.2.1
```
Comment (KristianKjerstad) [inline example/docker-compose.yaml:5]: treeView is broken in the latest dmss

File: packages/dm-core-plugins/package.json [hunk 3]
```diff
@@ -56,7 +56,8 @@
     "prebuild": "rm -rf dist",
     "copy-yaml-css-file": "cp src/yaml/index.css ./dist/yaml",
     "copy-explorer-css-file": "cp src/explorer/style.css ./dist/explorer",
-    "build": "tsc && yarn copy-explorer-css-file && yarn copy-yaml-css-file && yarn copy-explorer-css-file",
+    "copy-attribute-selector-icons-file": "cp src/attribute-selector/eds-icons.json ./dist/attribute-selector",
```
Comment (soofstad) [inline packages/dm-core-plugins/package.json:59]: Cant this be better handled by adding the file to "files" in package.json?

**Review by soofstad — APPROVED** [review]

**Pull request merged**